virtual_rainforest/models/abiotic/radiation.py

```diff
@@ -89,10 +94,17 @@ class Radiation:
     * sunshine_fraction: fraction of sunshine hours, between 0 (100% cloud cover)
         and 1 (cloud free sky), default = 1
 
-    PPFD and top-of-canopy radiation could be calculated in the AbioticModel __init__
-    for all timesteps provided in data. Something like a `calculate_radiation_balance`
-    function could then update the `longwave_radiation` and `netsurface_radiation`
-    attributes with each time step. This is currently not fully implemented.
+    Further, the class expects a dataclass that contains the following constants:
+    * cloudy_transmissivity: Cloudy transmittivity
+    * transmissivity_coefficient: Angular coefficient of transmittivity
+    * flux_to_energy: float From flux to energy conversion, umol J-1
+    * stefan_boltzmann_constant: Stefan-Boltzmann constant = 5.67e-8 W m-2 K-4
+    * soil_emissivity: Soil emissivity
+    * canopy_emissivity: Canopy emissivity
+    * beer_regression: Parameter in equation for atmospheric transmissivity based on
+        regression of Beer's radiation extinction function
+    * celsius_to_kelvin: Factor to convert temperature in Celsius to absolute
+        temperature in Kelvin= 273.15
```

Comment:
The API documentation for the `RadiationConstants` dataclass is going to show all of this information, so I'd keep it simple here and just say:

```rst
The ``const`` argument takes an instance of class
:class:`~virtual_rainforest.models.abiotic.radiation.RadiationConstants`, which provides
a user modifiable set of required constants.
```

That link then allows people to see the details and avoids breaking DRY.